### ISSUE TITLE
Check dispRity citation 

### ISSUE THREAD

**Issue by nhcooper123** [nhcooper123/time-slice#27]
I've cited v 0.5 as on zenodo for now...

**Comment by TGuillerme** [comment]
The version shouldn't matter as long as the following are exact (fir Google Scholar):

 * Title: dispRity: a package for measuring disparity in R
 * Date: 2015/10/2
 * DOI: 10.5281/zenodo.31742

The version numbers can be added but is probably not necessary since I'm going to move to 0.6 and 1.0 (with the MEE pub) probably soon.

**Comment by nhcooper123** [comment]
OK - the main difference is v 0.5 has me and Dave Bapst as authors... I can leave off the version number as you're right zenodo fixes that now. Hopefully by the time we get proofs there will be a paper to cite instead of the package

**Comment by TGuillerme** [comment]
Yeah anyways the ZENODO dois are while waiting for the MEE ones. Citations can be a bit dispersed until then but after that I'll have a proper easy to cite paper (hopefully!).
